kms/apiv1/options.go

```diff
@@ -130,6 +143,10 @@ type Options struct {
 
 	// Profile to use in AmazonKMS.
 	Profile string `json:"profile,omitempty"`
+
+	// StorageDirectory is the path to a directory to
+	// store serialized TPM objects. Only used by the TPMKMS.
+	StorageDirectory string `json:"storageDirectory,omitempty"`
```

Comment:
My preference is to remove all these extra options and have them configured through the URI. But I am ok to accepting this change.

Reply:
For programmatic configuration, this is a bit simpler to use. I figured a user of the package could be configuring this in their own program. Doing that through the URI is a bit more cumbersome than doing this. For the most common configurations, I think the property warranted. If we ever decide to remove it, users of the package will see the upgrade breaking their build, so I think it's OK for now.